.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
 *.core
 *.log
-.vscode
\ No newline at end of file
+.vscode
+*.ogg
```

Comment:
You also need to `git rm` the file you want to remove; just putting it in the gitignore isn't enough

Reply:
Done